Dockerfile.nvidia

```diff
@@ -20,7 +20,7 @@ WORKDIR $GOPATH/src/github.com/Mellanox/whereabouts/whereabouts
 COPY . .
 RUN ./hack/build-go.sh
 
-FROM nvcr.io/nvidia/distroless/go:v3.1.6
+FROM nvcr.io/nvidia/distroless/go:v3.1.8
```

Comment:
nvcr.io/nvidia/distroless/go:v3.1.10 is available.
Also need to backport to release branch